src/data/skill-desc.csv

```diff
@@ -1,447 +1,447 @@
+"レース終盤に前方が詰まると\nお告げ次第で道をわずかに切り開く","Slightly easier to find an opening if blocked at the front in Final Leg"
 "レース終盤のコーナーで\n中団以降のポジションから追い抜くと\nちょっと前を目指す",""
-"レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
-"レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
+"レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","When you're in 3rd place at the end of the race and about to lose, your speed slightly increases"
+"レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","After proceeding through the race calmly, your speed increases a little during the last 200m"
 "中盤で2回スキルを使うと\n持久力がわずかに回復する","Restore endurance slightly if you activate two skills in the middle part of the race"
-"最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
-"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","<size=22>In the race's second half, increase speed slightly when recovering endurance with skills while in front</size>"
-"中盤に後ろの方で順位を下げた時に\n持久力をわずかに回復する","<size=22>During Middle Leg, slightly recover stamina when overtaken in the back of the pack</size>"
-"最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる","<size=22>Slightly increase speed after the last corner when stamina runs out being one of the lead</size>"
-"最終コーナーで中団から前を狙う時\n加速力をわずかに上げる","<size=22>On last corner, slightly gain acceleration when aiming for the lead from the middle pack</size>"
-"レース中盤で前の方から\n仕掛け始める時に速度がわずかに上がる","<size=22>Slightly increase speed in Middle Leg when overtaking someone close to lead</size>"
-"最終コーナーで前の方で抜くと速度が後\nろの方で抜くと加速力がわずかに上がる","<size=22>On last corner, slightly gain speed when passing in front or acceleration when passing in back</size>"
-"レース後半に後方で詰め寄られると\n幸運パワーが身体に実り\n速度がわずかに上がる",""
+"最終直線で前の方で追いすがっていると\n速度がわずかに上がる","Slightly increase speed in the last straight when chasing someone close to the lead"
+"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","In the race's second half, increase speed slightly when recovering endurance with skills while in front"
+"中盤に後ろの方で順位を下げた時に\n持久力をわずかに回復する","During Middle Leg, slightly recover stamina when overtaken in the back of the pack"
+"最終コーナー以降に前めで力が尽きかけ\n詰め寄られると速度がわずかに上がる","Slightly increase speed after the last corner when stamina runs out being one of the lead"
+"最終コーナーで中団から前を狙う時\n加速力をわずかに上げる","On last corner, slightly gain acceleration when aiming for the lead from the middle pack"
+"レース中盤で前の方から\n仕掛け始める時に速度がわずかに上がる","Slightly increase speed in Middle Leg when overtaking someone close to lead"
+"最終コーナーで前の方で抜くと速度が後\nろの方で抜くと加速力がわずかに上がる","On last corner, slightly gain speed when passing in front or acceleration when passing in back"
+"レース後半に後方で詰め寄られると\n幸運パワーが身体に実り\n速度がわずかに上がる","Increase speed slightly when chasing the back of the pack in the final leg"
```

Comment:
I'm not sure how I missed this, but let's not remove the `<size>` markup in `skill-desc.csv`.  
Unfortunately most of them will definitely be cut-off without

Reply:
Since I wasn't originally planning to publicize this I got overzealous removing a bunch of size markup that didn't seem to work lol. That said, I haven't encountered any problems without it in the places I've seen, including the skills.
I'll revert this but could you point me to where it's failing without markup for reference?

Reply:
The markup does work, but sometimes it won't when the content is way too long and it gets cut off at exactly the markup closing tag I think?  
It should be easy to find text that are cut off without the size tag; just try to check the skill description of signature skills.